tail_test.go

```diff
@@ -471,8 +471,20 @@ func (t TailTest) StartTail(name string, config Config) *Tail {
 	return tail
 }
 
-func (t TailTest) VerifyTailOutput(tail *Tail, lines []string) {
+func (t TailTest) VerifyTailOutput(tail *Tail, lines []string, expectEOF bool) {
```

Comment:
Now that I actually look at the callers, having a bool argument sucks (because it's hard to tell what it is).  But the other option in go-land is to have a struct:
```go
tailTest.verifyTailOutput(tail, []string{"hello", "world"}, verifyTailOutputOptions{expectEOF: true})
```
And that's pretty terrible too. Sigh. Why, golang, why do you not have keyword/named arguments.

P.S. This should probably not be exported.